Arm/Hand/hand.hpp

```diff
@@ -41,17 +31,64 @@ class Hand
     pointer_.Initialize();
     thumb_.Initialize();
   }
-  void HandleHandMovement(float thumb_position,
-                          float pointer_position,
-                          float middle_position,
-                          float ring_position,
-                          float pinky_position)
+
+  void PrintHandData()
   {
+    printf("Hand Finger Positions:\n");
+    printf("Pinky Angle: %d\n", pinky_.GetPosition());
+    printf("Ring Angle: %d\n", ring_.GetPosition());
+    printf("Middle Angle: %d\n", middle_.GetPosition());
+    printf("Pointer Angle: %d\n", pointer_.GetPosition());
+    printf("Thumb Angle: %d\n", thumb_.GetPosition());
+
+    wrist_.PrintWristData();
+  }
+
+  void HandleHandMovement(
+      float speed,
+      float thumb_position,
+      float pointer_position,
+      float middle_position,
+      float ring_position,
+      float pinky_position, 
+      float roll_position, 
+      float pitch_position)
```

Comment:
At some point we should convert this to a struct since having a ton of parameters is bad practice

Reply:
I wanted to do this in a different PR, but will do.